cms/static/js/models/settings/course_grader.js

```diff
@@ -66,9 +66,10 @@ var CourseGrader = Backbone.Model.extend({
             else attrs.drop_count = intDropCount;
         }
         if (_.has(attrs, 'min_count') && _.has(attrs, 'drop_count') && !_.has(errors, 'min_count') && !_.has(errors, 'drop_count') && attrs.drop_count > attrs.min_count) {
-            errors.drop_count = _.template(
-                gettext("Cannot drop more <% attrs.types %> than will assigned."),
-                attrs, {variable: 'attrs'});
+            var template = _.template(
```

Comment:
Note to reviewers: the previous code was actually wrong, resulting in text of "Cannot drop more than will assigned.".